src/initial-submission/components/DetailsForm.tsx

```diff
@@ -21,6 +21,19 @@ interface Props {
     ButtonComponent?: (props: { saveFunction?: Function }) => JSX.Element;
 }
 
+const useSuggestions = (values: ManuscriptDetails, suggestions: Array<Suggestion>): ManuscriptDetails => {
+    const detail = (values ? values : {}) as ManuscriptDetails;
+    const titleIsBlank = !detail.title || detail.title === '';
+    const suggestedTitle = suggestions.reduce(
+        (title, item) => (title += item.fieldName == 'title' ? item.value : ''),
+        '',
+    );
+    if (titleIsBlank && suggestedTitle) {
+        detail.title = suggestions[0].value;
+    }
+    return detail;
+}
```

Comment:
Surprised the pre-commit didn't catch this. Have you got un-staged stuff that needs pushing?

Reply:
Doh!